services.html

```diff
@@ -0,0 +1,172 @@
+<!DOCTYPE html>
+
+<html lang="en">
+  <head>
+    <meta charset="utf-8" />
+
+    <title>Great Idea!- Services</title>
+
+    <link
+      href="https://fonts.googleapis.com/css?family=Bangers|Titillium+Web"
+      rel="stylesheet"
+    />
+    <link rel="stylesheet" href="./css/index.css" />
+
+    <!--[if lt IE 9]>
+      <script src="https://cdnjs.cloudflare.com/ajax/libs/html5shiv/3.7.3/html5shiv.js"></script>
+    <![endif]-->
+  </head>
+
+  <body class="container">
+    <div class="service head">
+      <nav>
+        <div class="service head links">
```

Comment:
Why would you have a div inside a nav?
It is redundant!
Move your class to the nav or alternatively you can add styles directly to the nav tag itself